src/foam/u2/tag/Input.js

```diff
@@ -135,6 +135,10 @@ foam.CLASS({
 
       this.initCls();
       this.link();
+      if ( this.autofocus ) {
```

Comment:
Why does Input have an 'autofocus' property? Don't all element's have a 'focused' property which causes .focus() to be called after they're loaded? If you remove this block and instead of setting autofocus: true below you set focused: true, does it still work?

Reply:
Do you want to remove 'autofocus' property? (You added it)
focused did work

Reply:
Updated to use focused